tailnet/conn.go

```diff
@@ -709,54 +711,34 @@ func (c *Conn) MagicsockServeHTTPDebug(w http.ResponseWriter, r *http.Request) {
 	c.magicConn.ServeHTTPDebug(w, r)
 }
 
+// Currently called when connected to the Agent with the given IP using the given application.
```

Comment:
Comments on any method (public or private) should be in the form `// FunctionName ...`
```suggestion
// SendConnectedTelemetry should be called when connection to a peer with the given IP is established.
```